### PR TITLE
TextField: Update mdc package to 0.26.0 and renamed helptext to HelperText

### PR DESCRIPTION
**Pull request by porfirioribeiro**
Closes #438

### CODE REVIEW

**Comment by prateekbh** [comment]
@porfirioribeiro closed?

**Comment by porfirioribeiro** [comment]
There are breaking changes in TextField helptext!

https://github.com/material-components/material-components-web/blob/master/CHANGELOG.md#0260-2017-11-27

**Comment by porfirioribeiro** [comment]
My bad, was cleaning up my repo and deleted this branch, sorry

**Comment by porfirioribeiro** [comment]
Fixed conflicts, please review @prateekbh

**Comment by mellogarrett** [comment]
It would be great if we allowed both the `helptext` & `helperText` prop, and just converted `helptext` to `helperText` behind the scenes while providing a deprecation warning. 

Thoughts?

**Comment by prateekbh** [comment]
hmmm not sure about it... shouldn't we just let it flow as `mdc-web` actually takes these decisions

**Comment by mellogarrett** [comment]
Just a thought. I don't feel strongly either way.

**Comment by porfirioribeiro** [comment]
It's harder for people to upgrade, but i think pmc should follow mdc-web
But it should be mentioned on a changes file, and link to original release
So pmc users can know what happened and why

**Comment by porfirioribeiro** [comment]
If we ever want to merge this branch i need to update the TypeScript definitions to match this changes!

**Comment by prateekbh** [comment]
so sorry, was relocating between countries, will get on speed now 👍

**Comment by porfirioribeiro** [comment]
Will do the TS changes and commit so we can merge this.

**Comment by prateekbh** [comment]
Thanks for covering this @porfirioribeiro

File: TextField/index.d.ts [hunk 1]
```diff
@@ -2,23 +2,23 @@ import MaterialComponent from '../MaterialComponent';
...
-declare class Helptext extends MaterialComponent<IHelptextProps, {}> {}
+declare class HelperText extends MaterialComponent<IHelperTextProps, {}> {}
 
 declare interface ITextFieldProps extends JSX.HTMLAttributes {
-  helptext?: string;
-  helptextPersistent?: string;
-  helptextValidationMsg?: string;
+  helperText?: string;
+  helperTextPersistent?: string;
+  helperTextValidationMsg?: string;
```
Comment (porfirioribeiro) [inline TextField/index.d.ts]: Maybe not for this PR but aren't we missing here some props @YePpHa ?
https://github.com/prateekbh/preact-material-components/blob/master/TextField/TextField.jsx#L125

Comment (porfirioribeiro) [reply TextField/index.d.ts]: Ok, PR closed 👍  
It can be taken care later

**Pull request merged**